vcbuild.bat

```diff
@@ -261,6 +261,7 @@ goto jslint
 
 :jslint
 if not defined jslint goto exit
+if not exist tools\eslint\bin\eslint.js goto exit
```

Comment:
thinking you should probably be as nice here as you are in the Makefile about what's wrong